
Header header

float32 Roll
float32 Pitch
float32 Yaw
float32 AccX
float32 AccY
float32 AccZ


string head
string timeUTC
string latitude
string northORsouth
string longitude
string eastORwest
string GPSstatus
string satelliteNum
string HDOP
string altitude
string altitudeUnit
string geiod
string geiodUnit
string rtkAge
string rtkID
int16 BCC
